src/main/scala/edu/ie3/simona/service/weather/WeatherSourceWrapper.scala

```diff
@@ -401,7 +401,7 @@ private[weather] object WeatherSourceWrapper extends LazyLogging {
       )
   }
   case object WeightSum {
-    val EMPTY_WEIGHT_SUM: WeightSum = WeightSum(0d, 0d, 0d, 0d)
+    val EMPTY_WEIGHT_SUM: WeightSum = WeightSum(1d, 1d, 1d, 1d)
```

Comment:
I think this is a mistake. The sum of weights should start at `0d`, respectively: Ideally, the sum should amount to `1d` _after_ summation. If weather data is missing for some coordinates, the calculated data divided by the sum of weights (which is then smaller than 1) should reproduce a realistic output.

But, since we're already here... I think the concept of using the class `WeightSum` is obsolete, since all data is weighted with the same weight for each coordinate. Which makes sense to me. So instead of using class `WeightSum`, we could just simply sum up all weights with a single variable.